lms/templates/courseware/courses.html

```diff
@@ -45,15 +65,36 @@
   </header>
 
   <section class="courses-container">
+
+    <div id="discovery-form">
+      <form>
+        <input class="discovery-input" placeholder="${_('Search for a course')}" type="text"/><!-- removes spacing
+        --><button type="submit" class="button postfix discovery-submit" aria-label="${_('Search')}">
+          <i class="icon fa fa-search" aria-hidden="true"></i>
+        </button>
+      </form>
+      <div id="discovery-message"></div>
+      <div id="loading-indicator" class="hidden">
```

Comment:
I like to wrap Loading indicators like this in a ```<div aria-live="polite" aria-relevant="all"></div>``` so that the loading state is announced to screen reader users.  aria-relevant="all" announces removals as well as addition, so the screen reader would say "Loading" and then wen the loading message is hidden again, it would say "Loading removed"  Not necessary, especially if you expect short load times, but its a nice touch.  The div with aria-live on it cannot ever be injected, or toggled hidden/visible.